File: program_data/ProgramData/30/395.cpp
//************************************
//????1000012825_3               *
//??????                       *
//?????2010.10.15               *
//????????7???????    *
//************************************
int main ()
{ 
	int sum = 0 , i , n = 1 ;                                                      // ???? ? ???
	cin >> i ;                                                                     // ?? i
	while (n <= i )                                                                // ????
	{
       if ( ( n % 7 == 0 ) ||  ( n % 10 == 7 ) || ( ( n > 69 ) && ( n< 80 ) )  )   // ??????7 ????
	      sum = sum ;
		  else
		  sum = sum + n * n ;                                                         // ??
	      n = n + 1 ;                                      
    }
	cout << sum << endl ;                                                          // ????
	return 0;
}

File: program_data/JavaProgramData/30/395.java
package <missing>;

public class GlobalMembers
{
	//************************************
	//????1000012825_3               *
	//??????                       *
	//?????2010.10.15               *
	//????????7???????    *
	//************************************
	public static int Main()
	{
		int sum = 0; // ???? ? ???
		int i;
		int n = 1;
		i = Integer.parseInt(ConsoleInput.readToWhiteSpace(true));
		while (n <= i) // ????
		{
		   if ((n % 7 == 0) || (n % 10 == 7) || ((n > 69) && (n < 80))) // ??????7 ????
		   {
			  sum = sum;
		   }
			  else
			  {
			  sum = sum + n * n; // ??
			  }
			  n = n + 1;
		}
		System.out.print(sum);
		System.out.print("\n");
		return 0;
	}
}

